[2023-03-29 15:39:06,028] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:23:15.177924+00:00 [queued]>
[2023-03-29 15:39:06,039] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:23:15.177924+00:00 [queued]>
[2023-03-29 15:39:06,039] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:39:06,039] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:39:06,039] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:39:06,053] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): merge_task> on 2023-03-29 15:23:15.177924+00:00
[2023-03-29 15:39:06,058] {standard_task_runner.py:52} INFO - Started process 238 to run task
[2023-03-29 15:39:06,061] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'merge_task', 'manual__2023-03-29T15:23:15.177924+00:00', '--job-id', '506', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpxhftjtr6', '--error-file', '/tmp/tmpseslttzi']
[2023-03-29 15:39:06,061] {standard_task_runner.py:80} INFO - Job 506: Subtask merge_task
[2023-03-29 15:39:06,117] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.merge_task manual__2023-03-29T15:23:15.177924+00:00 [running]> on host bc45bc710023
[2023-03-29 15:39:06,195] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=merge_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:23:15.177924+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:23:15.177924+00:00
[2023-03-29 15:39:06,201] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/mysql/connector/connection_cext.py", line 538, in cmd_query
    query_attrs=self._query_attrs)
_mysql_connector.MySQLInterfaceError: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF EXISTS data_merge' at line 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_merge.py", line 33, in merge
    cursor.execute("DROP IF EXISTS data_merge")
  File "/home/airflow/.local/lib/python3.7/site-packages/mysql/connector/cursor_cext.py", line 271, in execute
    raw_as_string=self._raw_as_string)
  File "/home/airflow/.local/lib/python3.7/site-packages/mysql/connector/connection_cext.py", line 541, in cmd_query
    sqlstate=exc.sqlstate)
mysql.connector.errors.ProgrammingError: 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF EXISTS data_merge' at line 1
[2023-03-29 15:39:06,212] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=data_merge_dag, task_id=merge_task, execution_date=20230329T152315, start_date=20230329T153906, end_date=20230329T153906
[2023-03-29 15:39:06,227] {standard_task_runner.py:97} ERROR - Failed to execute job 506 for task merge_task (1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'IF EXISTS data_merge' at line 1; 238)
[2023-03-29 15:39:06,274] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-03-29 15:39:06,352] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
